pkg/karmadactl/describe/describe.go

```diff
@@ -43,21 +43,24 @@ var (
 		prefixed with NAME_PREFIX.`)
 
 	describeExample = templates.Examples(`
+		# Describe a pod in Karmada control plane
+		%[1]s describe pods/nginx
```

Comment:
Can this example go with deployments? As usually no pods in the Karmada control plane?